src/net.cpp

```diff
@@ -1715,7 +1749,7 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
             return;
 
         // Add seed nodes if DNS seeds are all down (an infrastructure attack?).
-        if (addrman.size() == 0 && (GetTime() - nStart > 60)) {
+        if (addrman.size() == 0 && (GetTime() - nStart > FIXEDSEEDS_DELAY)) {
```

Comment:
This is a behavior change too, I do understand the logic to increase privacy for fresh nodes if DNS  are busy but how did you pick up this number ?

Reply:
This change isn't needed -- if addrman is going to end up empty, it will have been empty for each dns seed which means there would have been no delay in querying the seed.